.changeset/cyan-frogs-judge.md

```diff
@@ -0,0 +1,6 @@
+---
+"@ledgerhq/wallet-api-server": patch
+"@ledgerhq/wallet-api-docs": patch
```

Comment:
Hey it should probably be a minor bump for a new option and we don't really need to version the docs as we don't publish them
```suggestion
"@ledgerhq/wallet-api-server": minor
```